qt/aqt/tts.py

```diff
@@ -205,7 +213,9 @@ def _parse_voice_line(self, line: str) -> TTSVoice | None:
 
         original_name = m.group(1).strip()
         tidy_name = f"Apple_{original_name.replace(' ', '_')}"
-        return MacVoice(name=tidy_name, original_name=original_name, lang=m.group(2))
+        return MacVoice(
+            name=tidy_name, original_name=original_name, lang=m.group(2), available=None
```

Comment:
In case you're wondering why I didn't make `available` a default attribute: The non-default attributes of the child classes would appear after it in the `__init__` signature, making it invalid. It would work with `@dataclass(kw_only=True)`, but that would require Python 3.10.

Reply:
Interesting, I was not aware subclassed dataclasses behaved that way.